### PR TITLE
fix: progress indicator animation

### PR DESCRIPTION
**Pull request by corquaid**
Description
---
- [x] Fix `ProgressIndicator` animation behaviour when onboarding progress skips forward.

Motivation and Context
---
#328

How Has This Been Tested?
---
Screen Recording 2022-07-01 at 11 25 04

### CODE REVIEW

File: applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx [hunk 1]
```diff
@@ -4,29 +4,44 @@ import { StyledContainer } from './styles'
...
+  const [fillTwo, setFillTwo] = useState<number | undefined>(0.0001)
+  const [fillThree, setFillThree] = useState<number | undefined>(0.0001)
+  const [fillFour, setFillFour] = useState<number | undefined>(0.0001)
 
+  // Bar one logic
   useEffect(() => {
     if (overallFill) {
       if (overallFill <= 0.25) {
         setFillOne(overallFill * 4)
+        // setFillTwo(0.0001)
```
Comment (tarnas14) [inline applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx]: remove commented code

File: applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx [hunk 2]
```diff
@@ -4,29 +4,44 @@ import { StyledContainer } from './styles'
 import { ProgressIndicatorProps } from './types'
 
 const ProgressIndicator = ({ overallFill }: ProgressIndicatorProps) => {
-  const [fillOne, setFillOne] = useState<number | undefined>(0)
-  const [fillTwo, setFillTwo] = useState<number | undefined>(0)
-  const [fillThree, setFillThree] = useState<number | undefined>(0)
-  const [fillFour, setFillFour] = useState<number | undefined>(0)
+  const [fillOne, setFillOne] = useState<number | undefined>(0.0001)
```
Comment (tarnas14) [inline applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx]: I'd move 0.0001 to a const somewhere, which would explain why its not 0
REALLY_CLOSE_TO_ZERO_BECAUSE_XYZ

**Review by tarnas14 — CHANGES_REQUESTED** [review]

File: applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx (hunk 1 above)
Comment (corquaid) [reply applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx]: Done.

File: applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx (hunk 2 above)
Comment (corquaid) [reply applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx]: Done!

**Review by tarnas14 — APPROVED** [review]

**Pull request merged**